src/tests/multihost/alltests/test_rfc2307bis.py

```diff
@@ -0,0 +1,58 @@
+""" Automation of rfc2307bis
```

Comment:
Can this test be paced here :  https://github.com/SSSD/sssd/blob/master/src/tests/multihost/alltests/test_rfc2307.py   ??

Reply:
No, rfc2307 and rfc2307bis are different and are/should be tested differently.